### ISSUE TITLE
Custom thresholds for good/ok/poor coverage in the reports

### ISSUE THREAD

**Issue by v0vanec** [lemurheavy/coveralls-public#1537]
It looks like coverage levels are hardcoded and there's no way to change them for reports, i.e. coverage <=80% is red, <=90% is yellow and 90+% is green. It would be nice to configure custom thresholds per source directory (or at least per repo) and even exclude some directories from test coverage. e.g. I want to set >=70% as OK coverage (yellow) and 80+% as good coverage (green), how do I do that?

<img width="1094" alt="Screen Shot 2021-03-24 at 7 14 16 PM" src="https://user-images.githubusercontent.com/79227580/112399901-22850e00-8cd5-11eb-9096-db77815515c3.png">

**Comment by afinetooth** [comment]
@v0vanec you're correct, currently those levels (and colors) are hard-coded. There’s no way to change them at this time.

But there is a somewhat longstanding feature request for this, for the exact behavior you describe, in our backlog so I have added you as a plus-one.

Thanks.
